packages/cdk/resources/StaticContentBucket.ts

```diff
@@ -0,0 +1,116 @@
+import {Fn, RemovalPolicy} from "aws-cdk-lib"
+import {Role} from "aws-cdk-lib/aws-iam"
+import {CfnKey, Key} from "aws-cdk-lib/aws-kms"
+import {
+  BlockPublicAccess,
+  Bucket,
+  BucketAccessControl,
+  BucketEncryption,
+  CfnBucket,
+  CfnBucketPolicy,
+  ObjectOwnership
+} from "aws-cdk-lib/aws-s3"
+import {Construct} from "constructs"
+
+import {AllowCloudfrontGetObjectPolicyStatement} from "../policies/s3/AllowCloudfrontGetObjectPolicyStatement"
+import {AllowCloudfrontKmsKeyAccessPolicy} from "../policies/kms/AllowCloudfrontKmsKeyAccessPolicy"
+
+/**
+ * Resources for a static content S3 bucket
+
+ */
+
+export class StaticContentBucket extends Construct{
+  public readonly bucket: Bucket
+  public kmsKey: Key
+
+  public constructor(scope: Construct, id: string){
+    super(scope, id)
+
+    // Context
+    /* context values passed as --context cli arguments are passed as strings so coerce them to expected types*/
+    const allowAutoDeleteObjects: boolean = this.node.tryGetContext("allowAutoDeleteObjects") === "true"
+    const cloudfrontDistributionId: string = this.node.tryGetContext("cloudfrontDistributionId")
+
+    // Imports
+    const auditLoggingBucket = Bucket.fromBucketArn(
+      this, "AuditLoggingBucket", Fn.importValue("account-resources:AuditLoggingBucket"))
+
+    const deploymentRole = Role.fromRoleArn(
+      this, "deploymentRole", Fn.importValue("ci-resources:CloudFormationDeployRole"))
+
+    // Resources
+    const kmsKey = new Key(this, "KmsKey", {
+      enableKeyRotation: true,
+      removalPolicy: RemovalPolicy.DESTROY
+    })
+    kmsKey.addAlias("alias/StaticContentBucketKmsKey")
+
+    const bucket = new Bucket(this, "Bucket", {
+      encryption: BucketEncryption.KMS,
```

Comment:
can we add a bucketName to this (that includes the stack name) to make it easier in dev to work out which bucket is for which pull request